packages/react-devtools-shared/src/devtools/store.js

```diff
@@ -337,6 +344,10 @@ export default class Store extends EventEmitter<{|
     return this._supportsReloadAndProfile && this._isBackendStorageAPISupported;
   }
 
+  get unsupportedRendererVersionDetected(): boolean {
```

Comment:
Is it normal that we use getters vs generic functions?

Reply:
In the DevTools `Store` I guess it is. Any reason to prefer one over the other?

Reply:
In DevTools `Store` I guess it is. Any reason to prefer one over the other?

Reply:
I guess it's fine as this won't be used on unsupported browsers anyway.

Reply:
Nah, DevTools has the benefit of only being used in Chrome & Firefox